src/cpp/session/modules/SessionFind.cpp

```diff
@@ -517,17 +517,27 @@ class GrepOperation : public boost::enable_shared_from_this<GrepOperation>
       }
    }
 
-   Error completeFileReplace()
+   Error completeFileReplace(std::set<std::string>* pErrorMessage)
    {
       if (fileSuccess_)
       {
          if (!currentFile_.empty() &&
              !tempReplaceFile_.getAbsolutePath().empty() &&
              outputStream_->good())
          {
-            Error error = FilePath(currentFile_).testWritePermissions();
+             Error error;
+             // For Windows we ignore this additional safety check
+             // it will always fail because we have inputStream_ reading the file
+#ifndef _WIN32
```

Comment:
I think we should wrap the whole test in the `ifndef`; the rest of the code is a guaranteed no-op on other platforms since there will never be an error.

Reply:
Good point, I fixed this and found another situation where we weren't returning a proper error message if the move failed so I fixed that too.